priv/repo/migrations/20180815021708_remove_not_null_from_leader_in_group.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Cadet.Repo.Migrations.RemoveNotNullFromLeaderInGroup do
```

Comment:
As discussed, should also drop unique constraint on `leader_id` in `groups` table.